x/bank/v2/keeper/msg_server.go

```diff
@@ -0,0 +1,47 @@
+package keeper
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"cosmossdk.io/x/bank/v2/types"
+)
+
+var _ types.MsgServer = msgServer{}
+
+type msgServer struct {
+	*Keeper
+}
+
+// NewMsgServer creates a new bank/v2 message server.
+func NewMsgServer(k *Keeper) types.MsgServer {
```

Comment:
we should opt to write handlers to avoid networking bleeding into modules

Reply:
I think we need to update gogoproto then, gogoproto generates the msg server.

Reply:
There is no need for services with handlers.

Reply:
I know, I mean, look at the gogoproto generated `tx.pb.go`, it creates a grpc message server, and has grpc imports.

Reply:
🤡 nvm